packages/ui/src/components/ChannelSearch/SearchResults.tsx

```diff
@@ -23,10 +23,37 @@ export function SearchResults({
     if (!search.loading && search.hasMore) {
       search.loadMore();
     }
-  }, [search.loadMore, search.loading, search.hasMore]);
+  }, [search]);
 
   const isInitial = search.query === '';
 
+  const renderItem = useCallback(
+    ({ item: post }: { item: db.Post }) => {
+      return (
+        <View
```

Comment:
Probably want to break out this component and memoize the item and its function props. Also, the double handlers here tell me something funky is going on here -- would suggest disabled or pointerEvents none like above if that's what you're trying to achieve.